reconcile/test/test_queris.py

```diff
@@ -0,0 +1,82 @@
+from unittest import TestCase
+from unittest.mock import create_autospec, patch
+
+from reconcile import queries
+from reconcile.utils import gql
+
+from .fixtures import Fixtures
+
+PERMISSIONS_QUERY = """
```

Comment:
Is this different from the `PERMISSIONS_QUERY` in `queries.py`? If so, probably worth adding a comment, otherwise can it just be imported?

Reply:
It is the same! Thank you for point this out